Index: ./ssh/misc.c
===================================================================
RCS file: /cvs/src/usr.bin/ssh/misc.c,v
retrieving revision 1.146
diff -u -p -r1.146 misc.c
--- ./ssh/misc.c	28 Jan 2020 01:49:36 -0000	1.146
+++ ./ssh/misc.c	12 Feb 2020 03:08:06 -0000
@@ -1493,7 +1493,7 @@ static const struct {
 	{ "cs6", IPTOS_DSCP_CS6 },
 	{ "cs7", IPTOS_DSCP_CS7 },
 	{ "ef", IPTOS_DSCP_EF },
-	{ "le", IPTOS_DSCP_LE },
+	{ "le", 0x01 },
 	{ "lowdelay", IPTOS_LOWDELAY },
 	{ "throughput", IPTOS_THROUGHPUT },
 	{ "reliability", IPTOS_RELIABILITY },
